lib/tests/streamlit/scriptrunner/script_runner_test.py

```diff
@@ -520,8 +520,10 @@ def require_widgets_deltas(
 
     # If we get here, at least 1 runner hasn't yet completed before our
     # timeout. Create an error string for debugging.
-    err_string = "require_widgets_deltas() timed out after {}s ({}/{} runners complete)".format(
-        timeout, num_complete, len(runners)
+    err_string = (
```

Comment:
Just to prove I'm not just rubberstamping these things, here's the one place where Black formatted the code besides just changing the header :rofl: